magnolia-voting-app/src/main/java/com/example/JcrIsPollEditableRule.java

```diff
@@ -0,0 +1,26 @@
+package com.example;
+
+import info.magnolia.ui.api.availability.AvailabilityDefinition;
+import info.magnolia.ui.availability.rule.AbstractJcrRule;
+
+import javax.jcr.Item;
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import java.util.Calendar;
+
+public class JcrIsPollEditableRule extends AbstractJcrRule<JcrIsPollEditableRuleDefinition> {
+    public JcrIsPollEditableRule(AvailabilityDefinition availabilityDefinition, JcrIsPollEditableRuleDefinition ruleDefinition) {
```

Comment:
Change JcrIsPollEditableRule to IsPollEditableRule

Reply:
It's conflicting with my idea. I suggested to have named like this. Imagine that we are relying on JRC, however I'm not sure if someone or magnolia changed their mind regarding to JRC implementation. So by that case all we may have to support other implementation of database -> need to change file name which I don not like it. Also, prefix as JCR not a bad idea.

Reply:
Oki, keep this class name.